lib/internal/languages/languageEn.dart

```diff
@@ -162,69 +162,69 @@ class LanguageEn extends Languages {
   @override
   String get labelLicenses => "Licenses";
   @override
-  String get labelChooseColor => "Choose Color";
+  String get labelChooseColor => "Choose Colour";
   @override
   String get labelTheme => "Theme";
   @override
-  String get labelUseSystemTheme => "Use System Theme";
+  String get labelUseSystemTheme => "System";
   @override
   String get labelUseSystemThemeJustification =>
-    "Enable/Disable automatic Theme";
+    "Automatic Theme";
   @override
-  String get labelEnableDarkTheme => "Enable Dark Theme";
+  String get labelEnableDarkTheme => "Dark";
```

Comment:
@Clashkid155 If 179 should be reverted, why is "Dark" OK?
They are both under "Theme".

Reply:
Wasn't supposed to include this forget about it.